commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -125,6 +133,29 @@ public Runner createRunner() {
         isKeyManagerApiEnabled);
   }
 
+  private void logNetworkSpecInformation() {
+    final ForkSchedule forkSchedule = eth2Spec.getForkSchedule();
+    final Map<SpecMilestone, UInt64> milestoneSlotMap =
+        forkSchedule
+            .streamMilestoneBoundarySlots()
+            .collect(Collectors.toMap(TekuPair::getLeft, TekuPair::getRight));
+
+    final StringBuilder logString = new StringBuilder(String.format("Network: %s%n", network));
+    forkSchedule
+        .getActiveMilestones()
+        .forEach(
+            m -> {
+              final String specName = m.getSpecMilestone().name();
+              final UInt64 forkEpoch = m.getFork().getEpoch();
+              final UInt64 boundarySlot = milestoneSlotMap.get(m.getSpecMilestone());
+              logString.append(
+                  String.format(
+                      "Spec Name: %s, Fork Epoch: %s, First Slot: %s%n",
```

Comment:
A BIt unsure what the correct terminology should be here. Is it milestones or specs? Thought milestones would be the more correct answer but not sure.

Also, do we need the slot? Was thinking of something more succinct, perhaps we could just do something simple like Besu and print the name with the epoch? i.e. maybe something like this
`Using milestones PHASE0: 0, ALTAIR: 74240, BELLATRIX: 144896`

Reply:
The slot, while it can be determined from the epoch, was useful when debugging the recent issues since the requests contain the slot

Reply:
Good point. Was just trying to find a way to make it more succinct. But we can't do that after all.